tests/test_authentication.py

```
@@ -14,6 +14,7 @@
 from types import SimpleNamespace
 from app.config import USER_NAME, API_KEY
 from app.routes import login_required, login_attempts
+from app.models.user import User
```

Pylint (3.3.7)

[error] Unable to import 'app.models.user'

(E0401)